app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -129,17 +129,12 @@ class AuthRepositoryTest : ShouldSpec(
 			tested.updateUserAvatar(userPhoto) shouldBe profile
 		}
 
-		should("return Profile WHEN userProfile called") {
-			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
-
-			tested.userProfile() shouldBe profile
-		}
```

Comment:
Why this has been removed?

Reply:
Honestly, deleted by mistake. Thank you

Reply:
Then please convert PR to draft if there is still ongoing work on your side